Services captiveportal edit

sbeaver-netgate opened the pull request:
Ready for review

sbeaver-netgate committed:
Merge https://github.com/SjonHortensius/pfsense into bootstrap

sbeaver-netgate committed:
Merge https://github.com/SjonHortensius/pfsense into bootstrap

sbeaver-netgate committed:
Merge https://github.com/SjonHortensius/pfsense into bootstrap

sbeaver-netgate committed:
services_captiveportal_zones_edit.php Conversion complete

Ready for review

SjonHortensius commented on usr/local/www/services_captiveportal_zones_edit.php:
```diff
@@ -84,40 +84,36 @@
+
+$form = new Form(new Form_Button(
+	'submit',
+	gettext("Continue")
```
Form_\* calls don't need gettext or htmlspecialchars

SjonHortensius commented on usr/local/www/services_captiveportal_zones_edit.php:
```diff
@@ -84,40 +84,36 @@
+	'zone',
+	'Zone name',
+	'text'
+))->setHelp('
```
Please remove newline here

SjonHortensius commented on usr/local/www/services_captiveportal_zones_edit.php:
```diff
@@ -84,40 +84,36 @@
+$section->addInput(new Form_Input(
+	'descr',
+	'Zone description',
+	'text'
```
**text** is the default, so you can remove this line (on L105 too)

SjonHortensius commented on usr/local/www/services_captiveportal_zones_edit.php:
```diff
@@ -84,40 +84,36 @@
+	'Zone name',
+	'text'
+))->setHelp('
+Zone name. Can only contain letters, digits, and underscores (_).');
```
Please enforce this guideline by doing $input->setPattern. In this case the argument would be '[0-9a-z_]+'

http://www.w3.org/TR/2011/WD-html5-20110525/common-input-element-attributes.html#the-pattern-attribute

sbeaver-netgate committed:
Revised as suggested

Thanks!

It would be very helpful to have a phpdoc to document the Forms
framework.

I had not noticed the setPatter method :(

sbeaver-netgate committed:
Remove unneeded gettext
